io.catenax.batch/2.0.0/Batch.ttl

```diff
@@ -0,0 +1,180 @@
+    samm:characteristic :PartTypeInformationCharacteristic.
+:CatenaXIdTrait a samm-c:Trait;
+    samm:preferredName "Catena-X ID Trait"@en;
+    samm:description "Trait to ensure data format for Catena-X ID"@en;
+    samm-c:baseCharacteristic :UUIDv4;
+    samm-c:constraint :UUIDv4RegularExpression.
+:LocalIdentifierCharacteristic a samm-c:Set;
+    samm:preferredName "Local Identifier Characteristic"@en;
+    samm:description "A batch may have multiple attributes, which uniquely identify that batch in a specific dataspace (e.g. the manufacturer`s dataspace)"@en;
+    samm:dataType :KeyValueList.
+:ManufacturingCharacteristic a samm:Characteristic;
+    samm:preferredName "Manufacturing Characteristic"@en;
+    samm:description "Characteristic to describe manufacturing related data"@en;
+    samm:dataType :ManufacturingEntity.
+:PartTypeInformationCharacteristic a samm:Characteristic;
+    samm:preferredName "Part Type Information Characteristic"@en;
+    samm:description "The characteristics of the part type"@en;
+    samm:dataType :PartTypeInformationEntity.
+:UUIDv4 a samm:Characteristic;
+    samm:preferredName "UUIDv4"@en;
+    samm:description "A version 4 UUID is a universally unique identifier that is generated using random 32 hexadecimal characters."@en;
+    samm:dataType xsd:string.
+:UUIDv4RegularExpression a samm-c:RegularExpressionConstraint;
+    samm:preferredName "Catena-X Id Regular Expression"@en;
+    samm:description "The provided regular expression ensures that the UUID is composed of five groups of characters separated by hyphens, in the form 8-4-4-4-12 for a total of 36 characters (32 hexadecimal characters and 4 hyphens), optionally prefixed by \"urn:uuid:\" to make it an IRI."@en;
+    samm:see <https://datatracker.ietf.org/doc/html/rfc4122>;
+    samm:value "(^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$)|(^urn:uuid:[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$)".
+:KeyValueList a samm:Entity;
+    samm:preferredName "Key Value List"@en;
+    samm:description "A list of key value pairs for local identifiers, which are composed of a key and a corresponding value."@en;
+    samm:properties (:key :value).
+:ManufacturingEntity a samm:Entity;
+    samm:preferredName "Manufacturing Entity"@en;
+    samm:description "Encapsulates the manufacturing relevant attributes"@en;
+    samm:properties (:date [
+  samm:property :country;
+  samm:optional "true"^^xsd:boolean
+]).
+:PartTypeInformationEntity a samm:Entity;
+    samm:preferredName "Part Type Information Entity"@en;
+    samm:description "Encapsulation for data related to the part type"@en;
+    samm:properties (:manufacturerPartId :nameAtManufacturer :classification).
+:key a samm:Property;
+    samm:preferredName "Identifier Key"@en;
+    samm:description "The key of a local identifier. "@en;
+    samm:characteristic :KeyTrait;
+    samm:exampleValue "BatchID".
```

Comment:
example value does not match the regex constraint

.localIdentifiers[0].key	should match pattern "^(manufacturerId|batchId)$"	
"BatchID"

change example value to batchId to match

Reply:
done